src/interfaces/JsonRpcMethods.ts

```diff
@@ -3,7 +3,7 @@ export enum JSON_RPC_REQUEST_METHOD {
 	GET_STATUS = "get_status",
 	GET_ADDRESSES = "get_addresses",
 	SEND_OPERATIONS = "send_operations",
-	GET_BLOCKS = "get_block",
+	GET_BLOCKS = "get_blocks",
```

Comment:
Be carefule now this endpoints takes a vector instead of one block id.

Reply:
fixed, thank you!